nuitka/tools/gen_helper_code/BINARY_ADD.py

```diff
@@ -0,0 +1,33 @@
+
+
+import jinja2
+
+"""
+def BINARY_ADDITION(a, b):
+	if(isinstance(a,str)):
+		print("Not an integer") 
+	else:
+		if (type(int(a)))==int:
+			if(isinstance(a,float)):
+				print("Not an Integer")
+			else:
+				c = a+b
+				return c
+"""
+macro = ("{% macro add(a,b) %}"
```

Comment:
I would try to avoid macros, pure Python code seems more powerful and readable.

Reply:
This was about implementing it in pure jinja2 form that's why I included macro. If you think Python code is more powerful with jinja2 than what is the problem with this code https://github.com/Nuitka/Nuitka/pull/306